stage.py

```diff
@@ -589,6 +589,17 @@ def text(self, text, hightlight=False):
                 if self.row >= self.height:
                     self.row = 0
 
+    def measure(self, text, hightlight=False):
```

Comment:
I wonder if it would make more sense to simply return the dimensions in the ``text()`` call, since you are going to call it anyways, and it doesn't matter in what order you do it.

Reply:
This is a good idea- I think it did cross my mind (I could be inventing memories here), but I didn't want to make breaking changes.

Reply:
It won't break any existing code — in Python you can call a method and ignore its return value, and there won't be any errors.